### PR TITLE
You can now see paper on crates (with color!)

### LINKED ISSUE #26820: bounty manifests should have a sprite when attached to crate
## Description
to prevent bozos bringing bounties back to station and bozos selling non-bounties

should be for manifests specifically and not any paper, since order invoices are also yellow

### PR DESCRIPTION
**Pull request by Vermidia**
## About the PR

Paper is now visible on the sprites of crates, including color. (Other branch got a bit cooked).
Currently in 4 flavors: Paper, Bounty Paper, and the Captains Lavender Paper, and Invoice Paper (last one I kinda nabbed from tgs pallete)

## Why / Balance

Easier to see when something's been attached to a crate.
Closes #26820

## Technical details

See #26828, I implemented pretty much all of that, just changing the paperlabeltype to a string for easy of adding of more paper types without changing code

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
Lookatallthesenotes 
colour

## Breaking changes

**Changelog**

:cl:
- add: You can now see paper attached on crates. Color differs depending on the paper used.
- tweak: The labels on bodybags also have different colors depending on the paper used.
- tweak: Cargo Invoices are now blueish (to differentiate them from Cargo Bounties)

### CODE REVIEW

File: Content.Server/Labels/Label/LabelSystem.cs [hunk 1]
```diff
@@ -84,10 +84,7 @@ private void OnComponentInit(EntityUid uid, PaperLabelComponent component, Compo
         {
             _itemSlotsSystem.AddItemSlot(uid, ContainerName, component.LabelSlot);
 
-            if (!EntityManager.TryGetComponent(uid, out AppearanceComponent? appearance))
-                return;
-
-            _appearance.SetData(uid, PaperLabelVisuals.HasLabel, false, appearance);
+            UpdateAppearance((uid, component, null));
```
Comment (deltanedas) [inline Content.Server/Labels/Label/LabelSystem.cs:87]: dont need null for optional ones so just (uid, component)

File: Content.Server/Labels/Label/LabelSystem.cs [hunk 2]
```diff
@@ -131,10 +128,18 @@ private void OnContainerModified(EntityUid uid, PaperLabelComponent label, Conta
             if (args.Container.ID != label.LabelSlot.ID)
                 return;
 
-            if (!EntityManager.TryGetComponent(uid, out AppearanceComponent? appearance))
+            UpdateAppearance((uid, label, null));
```
Comment (deltanedas) [inline Content.Server/Labels/Label/LabelSystem.cs:131]: .

File: Content.Shared/Labels/Components/PaperLabelTypeComponent.cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+/// <summary>                                                                                                                                                 
+/// Specifies the paper type (see textures/storage/crates/labels.rsi to see currently supported paper types)  to show on crates this label is attached to.                                                                     
+/// </summary>                                                                                                                                                
+[RegisterComponent, NetworkedComponent]
+public sealed partial class PaperLabelTypeComponent : Component
+{
+    /// <summary>                                                                                                                                             
+    /// The type of label to show.                                                                                                                                    
+    /// </summary>
+    [DataField("paperType")]
```
Comment (deltanedas) [inline Content.Shared/Labels/Components/PaperLabelTypeComponent.cs:14]: replace with just `[DataField]` it autogenerates yaml name from c# name

File: Resources/Textures/Structures/Storage/Crates/labels.rsi/meta.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        {
+            "name": "paper"
+        },
+        {
+            "name": "bounty"
+        },
+        {
+            "name": "captains_paper"
+        }
+    ]
```
Comment (deltanedas) [inline Resources/Textures/Structures/Storage/Crates/labels.rsi/meta.json:19]: indent trolled use 2 spaces not 4

File: Resources/Textures/Structures/Storage/Crates/toybox.rsi/meta.json [hunk 5]
```diff
@@ -1,40 +1,49 @@
...
-  "license": "CC-BY-SA-3.0",
-  "copyright": "Made by brainfood118 (github) for ss14",
-  "size": {
-    "x": 32,
-    "y": 32
-  },
-  "states": [
-    {
-      "name": "crate"
+    "version": 1,
```
Comment (deltanedas) [inline Resources/Textures/Structures/Storage/Crates/toybox.rsi/meta.json:2]: untroll indent

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Comment by Vermidia** [comment]
I realized that while I did make the paper attachable, bounties and invoices still shared a color. So invoices are blueish now (can definitely change if it's too saturated/unfitting.

File: Resources/Prototypes/Entities/Structures/Storage/Crates/crates.yml [hunk 6]
```diff
@@ -571,6 +581,17 @@
...
+    layers:
+    - state: base
+    - state: closed
+      map: ["enum.StorageVisualLayers.Door"]
+    - state: welded
+      visible: false
+      map: ["enum.WeldableLayers.BaseWelded"]
+    - state: paper
+      sprite: Structures/Storage/Crates/labels.rsi
+      offset: "0.0,0.03"
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Structures/Storage/Crates/crates.yml:593]:
```suggestion
      offset: "0.0,0.03125"
```

needs to be multiples of the pixel size (1 / 32)

File: Resources/Prototypes/Entities/Structures/Storage/Crates/crates.yml [hunk 7]
```diff
@@ -581,6 +602,20 @@
...
+      map: ["enum.StorageVisualLayers.Door"]
+    - state: welded
+      visible: false
+      map: ["enum.WeldableLayers.BaseWelded"]
+    - state: locked
+      map: ["enum.LockVisualLayers.Lock"]
+      shader: unshaded
+    - state: paper
+      sprite: Structures/Storage/Crates/labels.rsi
+      offset: "0.0,0.03"
```
Comment (EmoGarbage404) [inline Resources/Prototypes/Entities/Structures/Storage/Crates/crates.yml:617]:
```suggestion
      offset: "0.0,0.03125"
```

here too

**Pull request merged**